internal/loggers/fields.go

```diff
@@ -33,10 +33,33 @@ func F(key string, value interface{}) Field {
 
 // CoordinateF builds a Field containing information taken from the provided coordinate
 func CoordinateF(coordinate coordinate.Coordinate) Field {
```

Comment:
What do you think to move this to subpackat `field`.
Then on caller side it can look like `field.Coordinate` or `field.Environment`.

Reply:
👍 done, great suggestion